pkg/attestlib/pkix_key.go

```diff
@@ -27,18 +27,38 @@ import (
 	"github.com/pkg/errors"
 )
 
+const rsaPkcs1Key = "RSA PRIVATE KEY"
+const ecPkcs1Key = "EC PRIVATE KEY"
+const pkcs8Key = "PRIVATE KEY"
+
 func parsePkixPrivateKeyPem(privateKey []byte) (interface{}, error) {
 	der, rest := pem.Decode(privateKey)
 
 	if len(rest) != 0 {
 		return nil, errors.New("expected one public key")
 	}
-
-	key, err := x509.ParsePKCS8PrivateKey(der.Bytes)
-	if err != nil {
-		return nil, err
+	switch der.Type {
+	case rsaPkcs1Key:
+		key, err := x509.ParsePKCS1PrivateKey(der.Bytes)
+		if err != nil {
+			return nil, err
```

Comment:
NIT: can you wrap the error? e.g., `errors.Wrap(err, fmt.Sprintf("parse %s key failed", der.Type))`. Same for below.

Reply:
done.